src/instructlab/configuration.py

```diff
@@ -302,7 +302,7 @@ class _serve_vllm(BaseModel):
     max_startup_attempts: int | None = None
     gpus: Optional[int] = None
     # arguments to pass into vllm process
-    vllm_args: list[str] | None = None
+    vllm_args: list[str] | None = []
```

Comment:
this is not fixing the issue - the type can still be null so you can have it set in config file.

Reply:
Btw this is also unsafe in python because then every instance of a config will reuse the same list object, so if you'd e.g. add something to it in runtime, other copies would immediately see it.